src/context_impl.cpp

```diff
@@ -353,6 +353,16 @@ void ContextImpl::initializeContextAttributes()
 	}
 }
 
+struct libm2k::IIO_CONTEXT_VERSION ContextImpl::getIioContextVersion()
```

Comment:
This method could be `const`. If there are issues due to `iio_context_get_version` not being const, one way of implementing this would be to have the `iioContextVersion` as a member of the `ContextImpl` class that would be set in the constructor. Then this method could simply return it.

Reply:
Modified.